src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchForm {
+  position: relative;
+  margin-top: 20px;
+}
+
+.bigSearchImage {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.bigSearchBar {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  font-family: 'Avenir', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #ffffff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigSearchBar::placeholder {
+  font-family: "Avenir", sans-serif;
+}
+
+.bigSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigSearchBar:focus {
+  font-family: "AvenirBold", sans-serif;
+  background: linear-gradient(180deg, #ffffff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.smallSearchForm {
+  position: relative;
+  margin-top: 20px;
+}
+
+.smallSearchImage {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.smallSearchBar {
+  padding-left: 33px;
+  width: 100%;
+  height: 42px;
+  line-height: 19px;
+  font-family: 'Avenir', sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #ffffff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
check out 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)